internal/xds/translator/route.go

```diff
@@ -314,7 +314,7 @@ func buildXdsURLRewriteAction(destName string, urlRewrite *ir.URLRewrite, pathMa
 		if urlRewrite.Path.FullReplace != nil {
 			routeAction.RegexRewrite = &matcherv3.RegexMatchAndSubstitute{
 				Pattern: &matcherv3.RegexMatcher{
-					Regex: "/.+",
+					Regex: "/.*",
```

Comment:
hey sorry for the delay here, lets tighten this up a bit with `^/.*$`

so something random like `blah/` doesnt get accepted